components/game/game.js

```diff
@@ -1,17 +1,21 @@
 import gameStyles from './game.module.scss';
 import { useState } from 'react';
 import Hand from "@/components/gameComponents/hand.js";
+import { dictionaryCombinations } from '@/components/utilities/combination.js';
```

Comment:
what's with the `@/components/...` nomenclature? i've never seen the `@` before a path, is this something specific to react/next?

Reply:
https://nextjs.org/docs/app/building-your-application/configuring/absolute-imports-and-module-aliases

- Next.js has built in way to handle relative paths

Reply:
ahhh nifty!